docs/docs/api/LayoutAnimations/EntryAnimations.md

```diff
@@ -0,0 +1,169 @@
+---
+id: entryAnimations
+title: Entering Animations
+sidebar_label: Entering Animations
+---
+
+## How to use predefined entering animation?
+
+### 1. Import chosen animation
+```js
+    import { Animation } from 'react-native-reanimated';
+```
+### 2. Choose Animated Component which entering you want to animate
+```js
+    <AnimatedComponent entering={Animation} >
```

Comment:
What is AnimatedComponent ? I think the case here is the same as above, we should say that you can use Animated.View as an example